docs/notebooks/example.ipynb

```diff
@@ -11,9 +11,9 @@
     "\n",
```

Comment:
<p>Again, I think <code>K</code> is more standard than <code>°K</code> (I get it might be for clarity, just pointing it out in case it's not intended)</p><p> </p><p>Also, there is a notebook on units, so maybe refer to it here too, as it's done in "usage.ipynb":</p><p> </p><p>&gt;&nbsp;The <a href="https://github.com/Ouranosinc/xclim/raw/47059a04ab7d1b236205864b1b59dbc87a78c135/docs/notebooks/units.ipynb" target="_blank">Units Handling</a> page goes more into detail on how unit conversion can easily be done.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xclim/pull/1338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>